examples/pylint/W0715_raising_format_tuple.py

```diff
@@ -0,0 +1,15 @@
+class DivisionException(Exception):
+    def __init__(self, message: str, numerator: int, denominator : int):
+        self.message = message
+        self.numerator = numerator
+        self.wasZeroDivide = (denominator == 0)
+        super().__init__()
+
+
+def divide(numerator: int, denominator: int) -> float:
+    """Divide the numerator by the denominator."""
+    try:
```

Comment:
This is a good idea, but you don't need a `try` here. Check if the `denominator` is 0, and if so do a `raise DivisionException(...)` the way that you have it below.

(`except` requires an exception class, not an exception instance, so the current example is misleading)